platformio.ini

```diff
@@ -72,43 +73,70 @@ lib_deps      =  ${all.lib_deps}
 board         = esp32dev
 monitor_speed = 115200
 upload_speed  = 921600
-upload_port   = COM10                    ; This will vary by machine and may not be needed if you have a single ESP attached
+upload_port   = /dev/ttyUSB0                        ; This will vary by machine and may not be needed if you have a single ESP attached
 build_flags   = -DDEMO=1
                 -std=gnu++17
+                -Dregister=                         ; Sinister:  redefine 'register' so FastLED can use that keyword under C++17
                 -Ofast 
 
-[env:wrover-demo]
-board           = esp-wrover-kit
-monitor_speed   = 115200
-upload_speed    = 1500000
-build_flags     = -DWROVERKIT=1
-                  -DDEBUG=1
-                  -DUSE_SCREEN=1
-                  -DUSE_PSRAM=0
-                  -DBOARD_HAS_PSRAM=0
-                  -std=gnu++17
-                  -Ofast 
-                  -mfix-esp32-psram-cache-issue
-build_type      = debug
-upload_port     = COM10
-debug_tool      = ftdi
-upload_protocol = ftdi
-debug_init_break = 
-;upload_protocol = ftdi
-
 [env:ledstrip]
 board         = heltec_wifi_kit_32
+build_type    = release
 monitor_speed = 115200
 upload_speed  = 921600
-upload_port   = COM10
-monitor_port  = COM10
-build_type    = release
+upload_port   = /dev/ttyUSB2
+monitor_port  = /dev/ttyUSB2
 build_flags   = -DLEDSTRIP=1
                 -DUSE_SCREEN=1
+                -DARDUINO_HELTEC_WIFI_KIT_32=1
+                -std=gnu++17
+                -DUNITY_INCLUDE_DOUBLE 
+                -DUNITY_DOUBLE_PRECISION=1e-12      ; Make doubles real 8 byte doubles
+                -Dregister=                         ; Sinister:  redefined 'register' so FastLED can use that keyword under C++17
+                -Os
+
+
+[env:chieftain]
```

Comment:
Do you think any of the new envs (chieftain, ledstriplite, umbrella, mesmerizer) should be included in the repo's CI?